### PR TITLE
Lowers ash walker hostility and prisoner crimes

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
rscadd: Ash walkers are not as hostile to outsiders as they once were. Perhaps our differences can be resolved without bloodshed.
/:cl:

### CODE REVIEW

**Comment by AlexCrimson** [comment]
booo

**Comment by KorPhaeron** [comment]
What's the point of this pr?

**Comment by coiax** [comment]
The point is people are complaining about "MUH GHOST ROLE FREE ANTAG", and I'm trying to deemphasise the antagonistic level of ashwalkers and prisoners, hopefully in the vein of de-antagging them

**Comment by KorPhaeron** [comment]
The entire point of ash walkers is for miners to have exciting targets to shoot though.

**Comment by Firecage** [comment]
This seems like a pointless PR honestly.

**Comment by coiax** [comment]
Well from the tone of OOC and #coderbus, apparently ash walkers are a cancer on the community, are free grief tickets, and the worst thing ever.

Maybe a tiny bit less hostility lore wise would be helpful.

**Comment by Zilenan91** [comment]
OOC is wrong and ashwalkers are a great addition

Most of the discourse that goes on there is one guy saying he doesn't like something and then some people who agree and half-agree echoing his statement

**Comment by YakumoChen** [comment]
what ever happened to the PR that gave Ashwalker IC reasons to stay on lavaland? As they are they're stronger then miners as they have long range spear throws (and embedding is a death sentence) and are often in larger groups then the starting miners.

Ashwalker is basically a free ticket to murderbone the station provided you can get enough access / window breaking to get onto it.

**Comment by KorPhaeron** [comment]
Using the word "free" as if most antag roles cost anything triggers me

**Comment by francinum** [comment]
It's free as in easy as hell to get.

**Comment by Zilenan91** [comment]
Just enable all antags and you get antag pretty much every other round. I don't see why it's a problem if people get to antag reliably, especially with all of the things going against ashwalkers.

**Comment by KorPhaeron** [comment]
Are other antag types difficult to get?

**Comment by francinum** [comment]
Let me clarify, it's guaranteed antag with a tacked on license to murderbone.

**Comment by KorPhaeron** [comment]
How is it gauranteed?

**Comment by francinum** [comment]
If the ruin spawns, they'll churn out egg after egg, so tack on near infinite respawning as well.

**Comment by KorPhaeron** [comment]
>garaunteed if they spawn

**Comment by francinum** [comment]
Which is from my experience pretty common.

**Comment by YakumoChen** [comment]
most antag rolls require you win a lottery at the start of the round or hope it's Rev/Double Rev/Cult. With Ash Ligger all you have to do is hit the Observe button and teleport to the ruin (which is fairly common to spawn) and you get a free (yes FREE) ticket to farm infinite grief tickets by hoarding eggs / collecting your FREE GUN then sending a mob of Antag Assistants to kill all the miners and murderbone the station.

**Comment by coiax** [comment]
So you need to win a different sort of lottery?

**Comment by YakumoChen** [comment]
Ash Liggers are the far worse problem because they can just farm eggs with the lavaland spawns and either reuse eggs for free respawns if they die or gather a small army of Greytide Liggers, and they're practically encouraged to murderbone the station once they kill the miners. 

Ash Liggers don't even need the ability to use guns. The regular station Greytide has always been able to do a disgusting amount of station damage with just stunprods and spears, Ash Liggers are no different, except that they get a License To Grief automatically.

Ash Liggers consistently ruin rounds once they're able to board the station and I've seen more then a fair share of rounds ruined by them boarding and murderboning everything they see, especially if they're able to find ruins loot. I distinctly remember one round where they almost overtook the entire station on top of zombies getting released because one of them found the banana staff and slipkilled everyone.

**Comment by KorPhaeron** [comment]
Rev
Cult
Clock Cult
Gang
Monkey/Shadowling/HoG (not in rotation)
Hacked as AI/your AI gets hacked as cyborg
Emagged as cyborg
Summon Guns
Revenant
Xenomorph
Swarmer
Apprentice/Demon
Necrostone
Soulstone
Stand
Syndieborg
Syndie reinforcements
Clockwork mobs
Xenobio creations
Blobbernauts
Most admin buttons pull from ghosts to replace or create antags

An incomplete list of """"free"""" antags that you get for either standing in a hallway or being dead.

SS13 is a game driven primarily by player conflict, it makes sense that people will end up in those roles.

**Comment by YakumoChen** [comment]
all of those are either mid-round antag ghost roles (which usually spawn late into the round to get a shuttle called) or already dependent on the current round type. Ash Ligger is available at roundstart and doesn't care what type of round it is, it can throw a huge wrench into things regardless.

Ash liggers are way too hard to deal with as miners because they have better range weapons, and embed is deadly, and they usually have better numbers.

**Comment by Firecage** [comment]
Sure, ash walkers might be a problem, but still, this PR is just silly and nitpicking on words. Like replaces murder with manslaughter.

**Comment by coiax** [comment]
Yes, because it's a role playing change. And small words can make big differences

**Comment by anconfuzedrock** [comment]
I would rather ass walkers be removed than this fucking bullshit
thanks to our lavaland rules I am going to kill any ash walker that tries to be friendly

**Comment by coiax** [comment]
> Changes some words suggesting that maybe you don't want to sacrifice every outsider

OMG THIS IS TOTALLY HUGBOX IMMA MURDER UUUUU U CAN'T TELL ME WHAT TO DO, IMMMA REBEEEEL

**Comment by Zilenan91** [comment]
If there think there will be no change then why did you make this?

**Comment by coiax** [comment]
I didn't say there would be no change? Changing people's attitudes to play has to start somewhere. May as well start in the flavour text.

File: code/game/objects/structures/ghost_role_spawners.dm [hunk 1]
```diff
@@ -228,8 +234,9 @@
 	L.name = L.real_name
 
 /obj/effect/mob_spawn/human/prisoner_transport/New()
-	var/list/crimes = list("murder", "larceny", "embezzlement", "unionization", "dereliction of duty", "kidnapping", "gross incompetence", "grand theft", "collaboration with the Syndicate", \
-	"worship of a forbidden deity", "interspecies relations", "mutiny")
+	var/list/crimes = list("manslaughter", "larceny", "embezzlement", "unionization", "dereliction of duty", "kidnapping", "gross incompetence", "grand theft", "collaboration with the Syndicate", \
+	"worship of a forbidden deity", "interspecies relations", "mutiny", \
+	"jaywalking", "littering", "hanging outside the brig suspiciously")
```
Comment (Cheridan) [inline code/game/objects/structures/ghost_role_spawners.dm]: add "banned for pooping"

Comment (francinum) [inline code/game/objects/structures/ghost_role_spawners.dm]: Failure to respond to inquiries due to deification.

**Comment by KorPhaeron** [comment]
>it can throw a huge wrench into things regardless

Strange interactions and things derailing are half the point of randomization though.

I've spaced the nuke as a free golem during war ops, and I've wiped out a tribe of ash lizards turned cultists as an alien hunter. Unplanned stories and adventures like that are what keep me playing, not a rigid meta routine.

**Comment by MMMiracles** [comment]
I wouldn't see an issue with ash walkers being an actual hostile tribe who fucked with people who fucked with their 'sacred lands', but as it stands there are people who actually jump between servers round-start in hopes of getting the better/existing lizard tribe for fucking over people.

I really wish I was joking when I said that but there are actually people doing this and have admitted to it in deadchat.

**Comment by PKPenguin321** [comment]
ITT: coiax got triggered by OOC calling his special snowflake lizards bad

also LINE BREAKS :-1: x :100:

**Comment by KorPhaeron** [comment]
I know at least two people have been banned recently for server hopping antag role/suicides

**Comment by InsaneHyena** [comment]
Absolutely disgusting. Do not merge under any circumstances.

**Comment by SidonCC** [comment]
This seems like what Kor intended Ashwalkers to be until cancerous faggots like rock ruined it. Why he isn't banned is beyond me.

**Comment by AlexCrimson** [comment]
Stop being babies. A few people were bitching about Ash Walkers in OOC along with Coiax because a few of them went on a murderbone on the escape shuttle. Next round this PR shows up. 

Its a neat feature. The problem is the shitty players that take any valid they can get and go wild with it.

**Comment by optimumtact** [comment]
None of our players actually use these descriptions to inform how they play @coiax so I don't see the point of this.

**Comment by optimumtact** [comment]
Also the fact that the ash lizards are murderboning is a matter for the policy subforum, I dont' think they were ever intended to be antags, but the admins are the ones treating them like they are

**Comment by Zilenan91** [comment]
If they weren't intended to be antags then why is their lore that they kill people and feed the bodies to an Eldritch tendril to reproduce and continue this cycle?

**Comment by optimumtact** [comment]
because it's just that, lore, which is not something our players ever take seriously, except apparently where it enables them to pretend they can be antags

**Comment by Zilenan91** [comment]
Their entire purpose is to hunt things down, then take the corpses back to their Eldritch tentacle and feed the corpses to it. After this, they bathe their gestating infants in the gore and then continue hunting for more and more things to continue reproducing to feed their sleeping God beneath the ground.

The only reason people don't take lore seriously is because almost all of it is shit or not there at all.

**Comment by coiax** [comment]
Closed for #18740

**Comment by optimumtact** [comment]
you know #18740 isn't getting merged right?

**Comment by coiax** [comment]
Speak for yourself

**Pull request closed without merging**